mergin/test/test_client.py

```diff
@@ -629,7 +638,8 @@ def get_project_info(mc, namespace, project_name):
     :param project_name: project's name
     :return: dict with project info
     """
-    projects = mc.projects_list(flag='created')
+    resp = mc.paginated_projects_list(flag='created')
+    projects = resp["projects"]
```

Comment:
I would suggest to keep the existing tests intact - and rather create a new test that would test the paginated projects list a bit. Otherwise the "old" function would stay untested...

Reply:
Done.